python_modules/dagster/dagster/_cli/workspace/cli_target.py

```diff
@@ -638,15 +614,15 @@ def get_repository_python_origin_from_kwargs(kwargs: ClickArgMapping) -> Reposit
     if kwargs.get("attribute") and not provided_repo_name:
         if kwargs.get("python_file"):
             _check_cli_arguments_none(kwargs, "module_name", "package_name")
-            python_file = unwrap_single_code_location_target_cli_arg(kwargs, "python_file")
+            python_file = kwargs["python_file"]
```

Comment:
another spot where a check would be useful so a code reader knows if this is string-only, tuple-only, or a dual state codepath